modules/config_import.py

```diff
@@ -0,0 +1,58 @@
+def get_config():
+    #Imports
+    import os
+    from configparser import ConfigParser
+    from configparser import ExtendedInterpolation
+    import json
+
+    #Get Config
+    conf = {}
+    os.chdir('config')
+    config = ConfigParser(interpolation=ExtendedInterpolation())
+    config.read('config.ini')
+    conf['OS.platform']           = config['OS']['platform']
+    conf['Cache.keep_cache']      = config['Cache']['keep_cache']
+    conf['Cache.cache_location']  = config['Cache']['cache_location']
+    conf['Search.search_local']   = ("True" == config['Search']['search_local'])
+    conf['Search.search_url']     = config['Search']['search_url']
+    conf['Remote.location']       = config['Remote']['location']
+    conf['Remote.branch']         = config['Remote']['branch']
+    conf['Remote.file_extension'] = config['Remote']['file_extension']
+    os.chdir('..')
+    return json.dumps(conf)
```

Comment:
Do you use json files to train or is there another reason?

Reply:
Json is easier to return and move as a value and I use it for other functions already in the rewrite. This is because the format is easier to parse and understand.